tensorflow/python/keras/engine/training.py

```diff
@@ -306,6 +306,9 @@ def compile(self,
         ValueError: In case of invalid arguments for
             `optimizer`, `loss`, `metrics` or `sample_weight_mode`.
     """
+    # NOTE: Argument `sample_weight_mode` is not used,
```

Comment:
Can we move this into the docstring for `sample_weight_mode`? I.e.

```
Arguments:
  ...
  sample_weight_mode: Unused argument. Note that passing this argument is no longer necessary. In `Model.fit`, 
   `sample_weight` can be either a 1-d Tensor for per-sample weighting, or a 2-d Tensor for per-sample-and-timestep 
       weighting.
```

Reply:
Good idea!